roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -1,50 +1,41 @@
 ---
+- name: Load lib_openshift modules
+  include_role:
```

Comment:
This include role should be moved into the meta/main.yml dependency list for openshift_manageiq role.  When inside of roles, dependencies should be listed in meta/main.yml.